Subdivision_method_3/include/CGAL/Subdivision_method_3/subdivision_masks_3.h

```diff
@@ -116,8 +118,8 @@ class Linear_mask_3 : public PQQ_stencil_3<PolygonMesh, VertexPointMap> {
   }
 
   void edge_node(halfedge_descriptor edge, Point& pt) {
-    const Point& p1 = get(this->vpmap, target(edge, *(this->pmesh)));
-    const Point& p2 = get(this->vpmap, source(edge, *(this->pmesh)));
+    const Point_ref p1 = get(this->vpmap, target(edge, *(this->pmesh)));
+    const Point_ref p2 = get(this->vpmap, source(edge, *(this->pmesh)));
```

Comment:
There are others that require the same changes in this file and also in `test_subdivision_method_3.cpp` and `Customized_subdivision_3.cpp`